tests/unit/onyxClearNativeStorageTest.js

```diff
@@ -73,15 +74,12 @@ describe('Set data while storage is clearing', () => {
         Onyx.clear();
         return waitForPromisesToResolve()
             .then(() => {
-                // Then the value in Onyx and the cache is the default key state
+                // Then the value in Onyx, the cache, and Storage is the default key state
```

Comment:
A nice side effect of this change: the cache and Storage are actually in sync when clearing.

Reply:
Love that